lab-kyler/test/note-route-test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const request = require('superagent');
+const expect = require('chai').expect;
+
```

Comment:
Make sure you require your server file into your test file, tests pass if you run the server separately but it's best practice to include it into your tests so you don't have to run tests and the server separately.